### DISCUSSION TITLE
Difficult with connecting crazyradio 2 with crazyflie

### DISCUSSION THREAD

**Discussion by Flavio20rt** [bitcraze/discussions#1688] [Q&A]
Good evening, 
Im working with some crazyflie 2.0.
I installed the client and the first days I made connection between radio and drone without problem and also made the First flight with the lighthouse deck, but now the crazyradio 2.0 can't connect to the drone , i can't understand why, I tried to change drivers, drone, radio, computer, use the bootloader mode, but I still have this problem. 
In these days I tried many times and it connected only two times, but I didn't change anything in the successful try.
In the last hours I tried but nothing happen.
Can you help me in any ways?

**Comment by ataffanel** [comment]
Can you try to flash the 1.1 version of the firmware from here: https://github.com/bitcraze/crazyradio2-firmware/releases/latest

It is possible you programmed your Crazyradio with the version 1.2 that was briefly released but had a major bug making the Crazyradio impossible to connect to after one connection.

You can put the Crazyradio 2 in bootloader mode by holding the button while inserting it in the USB port.
